# Request constants
int8 ONGROUND=1
int8 INAIR=2
# Request fields
int8 get_state
---
bool status

